src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/sms/AipAppellantNocRequestDecisionPersonalisationSms.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.appellant.sms;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.collect.ImmutableMap;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.NotificationType;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.SmsNotificationPersonalisation;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.service.RecipientsFinder;
+
+
+@Service
+public class AipAppellantNocRequestDecisionPersonalisationSms implements SmsNotificationPersonalisation {
+
+    private final String nocRequestDecisionAppellantSmsTemplateId;
+    private final RecipientsFinder recipientsFinder;
+
+    public AipAppellantNocRequestDecisionPersonalisationSms(
+        @Value("${govnotify.template.nocRequestDecision.appellant.sms}") String appealSubmittedAppellantSmsTemplateId,
+        RecipientsFinder recipientsFinder
+    ) {
+        this.nocRequestDecisionAppellantSmsTemplateId = appealSubmittedAppellantSmsTemplateId;
+        this.recipientsFinder = recipientsFinder;
+    }
+
+
+    @Override
+    public String getTemplateId() {
+        return nocRequestDecisionAppellantSmsTemplateId;
+    }
+
+    @Override
+    public Set<String> getRecipientsList(AsylumCase asylumCase) {
+        return recipientsFinder.findAll(asylumCase, NotificationType.SMS);
+    }
+
+    @Override
+    public String getReferenceId(Long caseId) {
+        return caseId + "_NOC_REQUEST_DECISION_APPELLANT_SMS";
+    }
+
+    @Override
+    public Map<String, String> getPersonalisation(Callback<AsylumCase> callback) {
+        requireNonNull(callback, "callback must not be null");
+
+        AsylumCase asylumCase =
+            callback
+                .getCaseDetails()
+                .getCaseData();
+
+        final String dateOfBirth = asylumCase
+            .read(AsylumCaseDefinition.APPELLANT_DATE_OF_BIRTH,String.class)
+            .orElseThrow(() -> new IllegalStateException("Appellant's birth of date is not present"));
+
+        final  String formattedDateOfBirth = LocalDate.parse(dateOfBirth).format(DateTimeFormatter.ofPattern("d MMM yyyy"));
```

Comment:
There is a default date formatter method in the implemented interface. Maybe you should use that to format the date of birth.